presto-main/src/main/java/com/facebook/presto/operator/WindowOperator.java

```diff
@@ -441,6 +452,7 @@ private void sortPagesIndexIfNecessary()
                 startPosition = endPosition;
             }
         }
+        windowInfo.addIndex(pagesIndex);
```

Comment:
This is part of a logic of window functions execution.
But if you insist, here's fixup: 049f1a508180e6fb0f02991a8cb45695934d3048.